bundler/spec/bundler/source/git/git_proxy_spec.rb

```diff
@@ -3,127 +3,177 @@
 RSpec.describe Bundler::Source::Git::GitProxy do
   let(:path) { Pathname("path") }
   let(:uri) { "https://github.com/rubygems/rubygems.git" }
-  let(:ref) { "HEAD" }
+  let(:ref) { nil }
+  let(:branch) { nil }
+  let(:tag) { nil }
+  let(:options) { { "ref" => ref, "branch" => branch, "tag" => tag }.compact }
   let(:revision) { nil }
   let(:git_source) { nil }
   let(:clone_result) { double(Process::Status, :success? => true) }
   let(:base_clone_args) { ["clone", "--bare", "--no-hardlinks", "--quiet", "--no-tags", "--depth", "1", "--single-branch"] }
-  subject { described_class.new(path, uri, ref, revision, git_source) }
+  subject(:instance) { described_class.new(path, uri, options, revision, git_source) }
```

Comment:
Nit: I know this is rubocop's preference but `instance` feels equally generic. Maybe `git_proxy` or just leave it as `subject` since it's already quite common in these tests. I don't mind merging this as is but if we're going to change it at all, maybe we can make it more clear.

Reply:
Amended the commit, and it is now named `git_proxy`.